debugcc.c

```diff
@@ -427,6 +448,11 @@ int main(int argc, char **argv)
 			if (clock_from_block(clk, block_name)) {
 				measure(clk);
 			}
+
+			if (clk->flags & IS_MCCC)
+				measure_mccc(clk);
+			else
+				measure(clk);
```

Comment:
No, see how Dmitry implemented a callback for this in #15 and how it was put to use in #17?